app/src/models/rebase.ts

```diff
@@ -42,3 +42,14 @@ export type RebasePreviewResult =
   | RebaseConflicts
   | RebaseUnsupported
   | RebaseLoading
+
+export type RebaseProgressSummary = {
+  /** A numeric value between 0 and 1 representing the rebase progress */
+  readonly value: number
```

Comment:
Same comment from before: https://github.com/desktop/desktop/pull/7117/files#r270187585

Reply:
Addressed in https://github.com/desktop/desktop/pull/7117/commits/322761dadb23373458ea6d0a729cde29326ebec6